src/components/Input/inputRadio/radio.js

```diff
@@ -67,7 +67,10 @@ export default class Radio extends Component {
                     tabIndex={tabIndex}
                     onFocus={onFocus}
                     onBlur={onBlur}
-                    onClick={onClick}
+                    onClick={event => {
+                        this.inputRef.current.focus();
+                        onClick(event);
+                    }}
```

Comment:
we should create a class method to handle this like:
```
handleClick(event) {
    if (isMozilla() || isSafari()) {
        this.inputRef.current.focus();
    }
    onClick(event);
}

...

onClick={this.handleClick}
```
also we need to add tests to this and note the helpers isMozilla and isSafari should be added to src/libs/utils

Reply:
for these helpers take a look here:
https://jsfiddle.net/6spj1059/
only add the needed ones